langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -33,6 +33,11 @@ public const float Infinity = 1.0/0.0;
 #
 # Exactly one of isFinite, isInfinite and IsNaN will be true for any float value
 #
+# ```ballerina
+# float finiteValue = 1.2;
+# boolean isFinite = finiteValue.isFinite();
```

Comment:
The example has to be generic, i.e., show how it works with an arbitrary float value. So I'd name the variable something like `floatValue`. If the value is known to be finite, there's no point actually calling `isFinite`, right?

Reply:
Applies to the other functions also.

Reply:
I prefer `f` in this context.  see https://go.dev/tour/moretypes/19. They are concise and easier the explain the point.

Reply:
Updated